pkg/agent/qrm-plugins/util/util.go

```diff
@@ -313,6 +315,18 @@ func GetContainerAsyncWorkName(podUID, containerName, topic string) string {
 	return strings.Join([]string{podUID, containerName, topic}, "/")
 }
 
+func GetSystemReservedCores(conf *config.Configuration, machineInfo *machine.KatalystMachineInfo, allCPUs machine.CPUSet) (machine.CPUSet, error) {
```

Comment:
what's the difference with `GetCoreNumsReservedForReclaim`

Reply:
There are two main differences:
1. `GetSystemReservedCores` returns certain CPU IDs but `GetCoreNumsReservedForReclaim` only returns the amount of cores.
2.  `GetSystemReservedCores` is related to the cores reserved for system_cores qos but `GetCoreNumsReservedForReclaim` is about the cores reserved for reclaimed_cores qos.

I'll rename it to `GetCoresReservedForSystem` to make it more consistent.